.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
+- [ ] Docstrings for all functions
```

Comment:
I understand.
Probably the best solution will be to keep the template as is for a while so to see which fields are actually used and which do cause obstacles.
Regarding the docsting guide: it should be better to address the detailed guide in `CONTRIBUTING.txt`.